udata_gouvfr/theme/templates/home.html

```diff
@@ -198,6 +198,59 @@ <h3 class="title ellipsis-dot">
       </div>
 </section>
 {% endif %}
+{% if forum_topics %}
+<section class="default forum-topics">
+      <div class="container forum-topics-container">
+            <div class="row">
+                <div class="col-xs-12">
+                    <h3><a href="{{ config.DISCOURSE_URL }}">{{ _('Latest forum discussions') }}</a></h3>
```

Comment:
Yes, but it has the same title semantics than all others sections on the page.
Table is mostly for layout here, not semantics